_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+  added in [coala webservices](https://gitlab.com/coala/landing/).
+* Discussion on improvements to be made in coala [community website](http://community.coala.io/) and
+  [projects website](http://projects.coala.io/)
+* Research on new models to be created (if any) and adding new fields to
+  existing db models in coala community website **db models**.
+* *Created* and *Merged* a cEP of how the new enhancements will be implemented
+  and how will it be beneficial to [coala community](https://github.com/orgs/coala/teams/coala-newcomers).
+
+#### CODING PHASE 1
+
+1. Implementation of methods in **coala [webservices](https://gitlab.com/coala/landing)**
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create new **[Community Logo](https://github.com/coala/community/issues/55) + Header Image of README.md**
+
+    b. Add information about new endpoints at coala-webserivces homepage.
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
+
+    d. Add [Organisation Teams](https://github.com/coala/community/issues/12)
+
+    e. Add [Mentors](https://github.com/coala/community/issues/11)
+
+    f. Add **Newcomer Invitation** [form](https://github.com/coala/community/issues/89#issuecomment-468955279)
+
+    g. Add **Promotion to Developer** [form](https://github.com/coala/community/issues/89)
+
+    h. Manage live data of community members
+
+    i. Addition of more data fields in **Users** db model
+
+    j. Add **Login and Sign-up** using GitHub
+3. Make Improvements (or fix bugs) in coala [community website](https://community.coala.io/)
+
+    a. Build failure for more than one remote [#235](https://github.com/coala/community/issues/235)
+
+    b. Netlify build failure for fork [#232](https://github.com/coala/community/issues/232)
+
+#### CODING PHASE 2
+
+1. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create **Profile Page**
+
+    b. Leaderboard on **[Homepage](https://coala-community.netlify.com/)**
+
+    c. Enhancement in **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+2. Implementation of listed enhancements in [projects website](https://projects.coala.io)
+
+    a. Add **[Filters](https://github.com/coala/projects/issues/559)** for searching projects
+
+3. Make Improvements in coala [community website](https://community.coala.io/)
+
+    a. **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+    b. **[UI](https://coala-community.netlify.com/)**
+      * [Re-designing of the website](https://github.com/coala/community/issues/44) and making it responsive.
+      * [Inactive Issue](https://coala-community.netlify.com/static/inactive-issues.json) and [unassigned issues activity](https://coala-community.netlify.com/static/unassigned-issues.json) web pages shows
```

Comment:
I am not sure if we need these functionalities in the website, these were only some temporary requirements iirc. @jayvdb

Reply:
Ohh, but I think it will be OK if we display them. What I was thinking to display only those issues that have importance medium or above it. It will allow open-source enthusiasts to get familiar with current important issues and a way can be provided to redirect to GitHub that will show all issues(Inactive or unassigned acc. to selection) of all levels.